src/scripts/upload_image_captioning.py

```diff
@@ -0,0 +1,16 @@
+"""Upload the image captioning dataset to the HuggingFace Hub.
+
+Usage:
+    >>> python src/scripts/upload_image_captioning.py
+"""
+
+
+from datasets import load_dataset
+
+DATASET_PATH = (
+    "../../../../../mnt/data_6tb/oliver/NordjyllandNews/data/processed/images/train"
+)
+DATASET_HF_PATH = "alexandrainst/nordjylland-news-image-captioning"
+
+dataset = load_dataset("imagefolder", data_dir=DATASET_PATH, split="train")
+dataset.push_to_hub(DATASET_HF_PATH, private=True)
```

Comment:
It's generally good practice to wrap a script inside a main function, and just call that main function if the script is called from the terminal. This is to avoid the case where this script is merely imported, in which case all of this code would be run.

Further, `mnt` lies in the root folder, so we can remove all the trailing `../..` and just start the path with a slash to indicate that it's in the root.

These can both be done as follows:

```suggestion

from datasets import load_dataset

DATASET_PATH = (
    "/mnt/data_6tb/oliver/NordjyllandNews/data/processed/images/train"
)
DATASET_HF_PATH = "alexandrainst/nordjylland-news-image-captioning"

def main() -> None:
    dataset = load_dataset("imagefolder", data_dir=DATASET_PATH, split="train")
    dataset.push_to_hub(DATASET_HF_PATH, private=True)
    
if __name__ == "__main__":
    main()
```